internal/models/alert_rule.go

```diff
@@ -18,19 +18,31 @@ type AlertingRule struct {
 
 	ScheduledTaskID *string
 	ExecutionStatus AlertingRuleExecutionStatus
+	AlertDelay      *AlertingRuleAlertDelay
 }
 
 type AlertingRuleSchedule struct {
 	Interval string
 }
 
+type AlertingRuleAlertDelay struct {
+	Active float32
+}
+
 type AlertingRuleAction struct {
-	Group  string
-	ID     string
-	Params map[string]interface{}
+	Group     string
+	ID        string
+	Params    map[string]interface{}
+	Frequency AlertingRuleActionFrequency
```

Comment:
It seems we don't use it.

Reply:
This will come on a follow up PR where I add more fields. I can remove it for now 👍